core/gui/src/app/workspace/service/workflow-computing-unit/workflow-computing-unit-managing.service.ts

```diff
@@ -30,9 +31,10 @@ export class WorkflowComputingUnitManagingService {
     cpuLimit: string,
     memoryLimit: string,
     gpuLimit: string = "0",
+    jvmMemorySize: string = "1G",
```

Comment:
should this be 2G?

Reply:
Initially it is set to 2G

```typescript
// Set JVM memory size to 1GB when CU memory is 1GB, otherwise set to 2GB
      if (cuMemoryInGb === 1) {
        this.jvmMemorySliderValue = 1;
        this.selectedJvmMemorySize = "1G";
      } else {
        // For 2-3GB instances, use 2GB for JVM
        this.jvmMemorySliderValue = 2;
        this.selectedJvmMemorySize = "2G";
      }
```

if the total memory == 1, then jvmMemorySize is 1GB
if the total memory >= 2, then jvmMemorySize(default) is 2GB